modules/ROOT/pages/hdfs/hdfs-connector.adoc

```diff
@@ -81,51 +83,42 @@ To use the HDFS connector in your Mule application, you must configure a global
 |Field |Description
 |NameNode URI |The URI of the file system to connect to.
 
-This is passed to the HDFS client as the FileSystem#FS_DEFAULT_NAME_KEY configuration entry. It can be overridden by values in configurationResources and configurationEntries.
+This is passed to the HDFS client as the FileSystem#FS_DEFAULT_NAME_KEY configuration entry. It can be overridden by values in *Configuration Resources* and *Configuration Entries*.
 |Username | User identity that Hadoop uses for permissions in HDFS.
 
 When Simple Authentication is used, Hadoop requires the user to be set as a System Property called HADOOP_USER_NAME. If you fill this field then the connector sets it for you, however you can set it by yourself. If the variable is not set, Hadoop uses the current logged in OS user.
 |Configuration Resources |A list of configuration resource files to be loaded by the HDFS client. Here you can provide additional configuration files. For example, core-site.xml.
-|Configuration Entries |A map of configuration entries to be used by the HDFS client. Here you can provide additional configuration entries as key/value pairs.
+|Configuration Entries |A map of configuration entries to be used by the HDFS client. Here you can provide additional configuration entries as key and value pairs.
 |===
 
 image::hdfs/hdfs-config.png[]
 
-
 === Kerberos Authentication Configuration
 
 [%header,cols="50a,50a"]
 |===
 |Field |Description
-|NameNode URI |The URI of the file system to connect to.
-
-This is passed to HDFS client as the FileSystem#FS_DEFAULT_NAME_KEY configuration entry. It can be overridden by values in configurationResources and configurationEntries.
-|Username | Kerberos principal.
-
-This is passed to HDFS client as the "hadoop.job.ugi" configuration entry. It can be overridden by values in configurationResources and configurationEntries. If not provided, it uses the currently logged in user.
-|KeytabPath |Path to the keytab file associated with username.
-
-KeytabPath is used in order to obtain TGT from "Authorization server".  If not provided it looks for a TGT associated to username within your local kerberos cache.
+|NameNode URI |The URI of the file system to connect to. This is passed to HDFS client as the FileSystem#FS_DEFAULT_NAME_KEY configuration entry. It can be overridden by the values in *Configuration Resources* and *Configuration Entries*.
+|Username | Kerberos principal. This is passed to HDFS client as the hadoop.job.ugi configuration entry. It can be overridden by the values in *Configuration Resources* and *Configuration Entries*. If not provided, it uses the currently logged in user.
```

Comment:
"the HDFS" client. Should hadoop.job.ugi be in backticks? 
What does "it" refer to in the last sentence?